source/features/batch-open-issues.tsx

```diff
@@ -39,7 +39,7 @@ function init() {
 			<button
 				type="button"
 				onClick={openIssues}
-				class="float-left btn-link rgh-open-all-selected"
+				className="float-left btn-link rgh-open-all-selected"
```

Comment:
This is because of React types. I read the `dom-chef` docs, and it supports both, correct @bfred-it? This change appears in other files below as well, so I'll just mention it once.

Reply:
We can do this for now, but if there's a way to support `class` it would be better (e.g. by extending the types?)

Reply:
Yeah, for sure. I mention it in comments in the global.d.ts file in my latest commit. I'm trying to figure out the best way to handle this. It's actually the most difficult part of this PR.

Reply:
Basically I'm trying to avoid rewriting the React types for JSX ;)